libs/@guardian/source-foundations/src/index.test.ts

```diff
@@ -27,6 +27,22 @@ it('Should have exactly these exports', () => {
 	expect(Object.keys(pkgExports).sort()).toEqual([
 		'FocusStyleManager',
 		'appearance',
+		'article15',
+		'article15Object',
+		'article17',
+		'article17Object',
+		'articleBold15',
+		'articleBold15Object',
+		'articleBold17',
+		'articleBold17Object',
+		'articleBoldItalic15',
+		'articleBoldItalic15Object',
+		'articleBoldItalic17',
+		'articleBoldItalic17Object',
+		'articleItalic15',
+		'articleItalic15Object',
+		'articleItalic17',
+		'articleItalic17Object',
```

Comment:
as with kicker below, should these be in source or dcr?

Reply:
As above, these have also been removed for now whilst awaiting guidance.